City-of-Helsinki/helsinki-paatokset
# PP-368: Mobile fixes

**To test**
- Checkout branch, run `make drush-cim drush-cr make ahjo-migrations`
- Run the following command in the container: `drush ap:update meetings U54010020228 -v`
- Check that there's a new "Frontpage calendar notice text" setting in the default texts: https://helsinki-paatokset.docker.so/fi/admin/tools/default-texts (under default fields)
  - Add "Kokouksen esityslista ilmestyy yleensä kokousta edeltävänä torstaina" as the default value
  - Make sure the field is translatable
  - Check that the text appears on the frontpage under the meetings calendar:  https://helsinki-paatokset.docker.so/fi/
- Open the meetings calendar page: https://helsinki-paatokset.docker.so/fi/kokouskalenteri
  - Check the mobile layout. It should no longer have extra padding on the left
- Check a trustee page (for example: https://helsinki-paatokset.docker.so/fi/paattajat/arhinpa)
  - Add some custom values and make sure they have initiatives listed
  - Test the mobile layout, there should be no extra left or right padding in the header ("valtuutettu , valtuustoryhmä") or in the accordions
- Check this document: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunkiymparistolautakunnan-rakennusten-ja-yleisten-alueiden-jaosto/asiakirjat/U54010020228
  - Test the mobile layout. The breadcrumb link should be properly truncated instead of breaking the layout
